Ginger/GingerCoreCommon/Run/RunSetConfig.cs

```diff
@@ -102,32 +102,32 @@ public string SealightsTestStage
             }
         }
 
-        private string mSealighsLabId;
+        private string mSealightsLabId;
         [IsSerializedForLocalRepository]
-        public string SealighsLabId
+        public string SealightsLabId
```

Comment:
changing the serialized field will break the old xml files. 
e.g. if you create solution on old version and try to open this xml on this version it will not be able to recognize the field. we have a mechanism to handle this kind of scenario, please confirm if this is taken care ?